.travis.yml

```diff
@@ -0,0 +1,29 @@
+os: linux
+
+sudo: enabled
+
+language: python
+
+python:
+  - "2.7"
+  - "3.6"
+
+before_install:
+  # We need to install python-3.6 from a third party repo as it is not
+  # available as a standard package on Travis Trusty VMs.
+  - sudo add-apt-repository ppa:jonathonf/python-3.6 -y
+  - sudo apt-get update -q
+  - sudo apt-get install python3.6
+  - sudo apt-get install python3.6-dev
+
+install:
+  - pip install networkx
+  - pip install six
+  - pip install pytype
+
+# Use one long command so that the build will fail on the first error:
+# https://github.com/travis-ci/travis-ci/issues/1066.
+script: |
+  pytype -V2.7 . --exclude tests/ testdata/ &&
```

Comment:
should this be exclude tests/,testdata/?

Reply:
This is the right format; --exclude uses nargs.